src/main/resources/templates/start.html

```diff
@@ -23,119 +23,123 @@
 </head>
 <body>
 <!--#region Package information -->
-<h4 id="packageInfoHeader">Package Information</h4>
-<!--Carrier select dropdown-->
-<div class="input-group mb-3" id="carrierSelect">
-    <div class="input-group-prepend">
-        <label class="input-group-text" for="carrierSelect">Carrier</label>
+<div id="packageInfoDiv">
```

Comment:
Added divs to Package Info, Starting Address, and Delivery Address for better organization. (There is too much on the screen at once, you can use these divs to hide what is unnecessary at the time and unhide the divs as they become necessary.)

Reply:
Good way to organize things.